src/electronic-ids/pcsc/EIDIDEMIA.cpp

```diff
@@ -69,7 +69,7 @@ void EIDIDEMIA::selectADF2() const
 byte_vector EIDIDEMIA::getCertificateImpl(const CertificateType type) const
 {
     selectMain();
-    return electronic_id::getCertificate(*card, type.isAuthentication() ? AUTH_CERT : SIGN_CERT);
+    return readFile(*card, type.isAuthentication() ? AUTH_CERT : SIGN_CERT, 0xC0);
```

Comment:
Perhaps add a comment that Idemia does not support 256 byte block length, need to limit the block length to 0xC0.